### ISSUE TITLE
run

### ISSUE THREAD

**Issue by lzzlxxlsz** [tSchlegl/f-AnoGAN#2]
what should I do for this project? which code should I run ?and What is the order of this code?

**Comment by alexofficial** [comment]
Hey, @lzzlxxlsz. You can run f-Anogan as follows:

Step1: Set image paths 

Step2: run `python wgangp_64x64.py`

Step3: Open the python file **z_encoding_izif.py** and add at the end of the file the path of the trained model.  **Example**: 
> train_enc_izi("./f-AnoGAN/wganTrain/wgangp_64x64_changed_normal_crIt5_2019-05-22-1043/checkpoints/", 7, loss_type='MSE', kappa=1.0)

Step 4: run `python z_encoding_izif.py`. After that, you will be able to see a folder named **wganTrain-iter7** 

Step 5: Do the same for anomaly_detection.py. **Example**:
> anomaly_scoring("../f-AnoGAN/wganTrain-iter7/wgangp_64x64_changed_normal_crIt5_2019-05-06-1943-enc_iter50000_2019-05-07-0912/z_encoding_izif_changed-MSE-k1.00_tanh_fc/checkpoints", 200)

Step 6: run `python anomaly_detection.py`. Then you can find inside: **wganTrain-iter7/../../_dil**  folder, a subfolder ( **_dil**) which contains the final results.

**Comment by lzzlxxlsz** [comment]
have you run the code? and the what is the dataset of yours?

**Comment by alexofficial** [comment]
> 
> 
> have you run the code? and the what is the dataset of yours?

Yes, I have my own dataset (64x64 pixel images). You have just to modify **_img_loader.py_** and set the paths.

**Comment by lzzlxxlsz** [comment]
How does the result?

**Comment by lzzlxxlsz** [comment]
in the step3, add the trained model.does the model need to be download?

**Comment by alexofficial** [comment]
> 
> 
> in the step3, add the trained model.does the model need to be download?

No, you do not need to download anything. After running `python wgangp_64_64.py` you will see a folder named **wganTrain**. Then, you just set the path of checkpoints folder which is located inside **wganTrain**.

**Comment by lzzlxxlsz** [comment]
my conmputer environment is cpu ,does this code suit it?thanks a lot

**Comment by tSchlegl** [comment]
Hi @lzzlxxlsz!
I strongly suggest to use a GPU for the sake of runtime.

@alexofficial, thanks for assisting lzzlxxlsz

**Comment by lzzlxxlsz** [comment]
thank you very much .and how does your result? does it find the abnomal area? and how about your dataset ? is there difference between them? my dataset is a little complex,just like the cat,but each picture is different but they are long to cat kind

**Comment by lzzlxxlsz** [comment]
excuse me ,I just want to ask you that my copmter environment is cpu ,and do I need change the setting of the code?

**Comment by lzzlxxlsz** [comment]
> Hi @lzzlxxlsz!
> I strongly suggest to use a GPU for the sake of runtime.
> 
> @alexofficial, thanks for assisting lzzlxxlsz

sorry，I have some questions ,the frst stage ,modify the image_loder.py,my dataset is 496x496,do I need modify the size?

**Comment by lzzlxxlsz** [comment]
> Hey, @lzzlxxlsz. You can run f-Anogan as follows:
> 
> Step1: Set image paths
> 
> Step2: run `python wgangp_64x64.py`
> 
> Step3: Open the python file **z_encoding_izif.py** and add at the end of the file the path of the trained model. **Example**:
> 
> > train_enc_izi("./f-AnoGAN/wganTrain/wgangp_64x64_changed_normal_crIt5_2019-05-22-1043/checkpoints/", 7, loss_type='MSE', kappa=1.0)
> 
> Step 4: run `python z_encoding_izif.py`. After that, you will be able to see a folder named **wganTrain-iter7**
> 
> Step 5: Do the same for anomaly_detection.py. **Example**:
> 
> > anomaly_scoring("../f-AnoGAN/wganTrain-iter7/wgangp_64x64_changed_normal_crIt5_2019-05-06-1943-enc_iter50000_2019-05-07-0912/z_encoding_izif_changed-MSE-k1.00_tanh_fc/checkpoints", 200)
> 
> Step 6: run `python anomaly_detection.py`. Then you can find inside: **wganTrain-iter7/../../_dil** folder, a subfolder ( **_dil**) which contains the final results.
Can I add you up？my wechat is 18731186906 ，I have some questions and need your help,thanks a lot

**Comment by choiwooyeon1** [comment]
Are u ok with cpu?
If u see this comment, Plz answer me 
That will help me a lot
